src/utils/constants.js

```diff
@@ -103,7 +105,21 @@ export const ALERT_MESSAGE = {
   [ALERT_ERROR_UNEXPECTED]: 'Therer was an unexpected error.',
 };
 
-export const ALERT_MESSAGE_DURATION = 3000;
+export const USER_MOUNTS_STEPPER_STEPS = [
+  "Selecting a user",
+  "Editing owned mounts",
+  "Saving or discarding changes",
+  "Selecting an expansion",
+  "Searching for a user",
+];
+
+export const USER_MOUNTS_STEPPER_INSTRUCTIONS = {
+  0: "",
+  1: "",
+  2: "",
+  3: "",
+  4: "",
```

Comment:
I don't know what this is for.

Reply:
This is part of an alternative styling option for help. Writing here fills in a Typography block above the gif display. It can be used to add more detailed explanations but it also breaks up the flow of the page a little.

Reply:
Obsolete design, removed